Fitbit.Portable/Models/DateRangePeriod.cs

```diff
@@ -19,12 +20,16 @@ public enum DateRangePeriod
         OneWeek,
         [StringValue("1m")]
         OneMonth,
+        [Obsolete("No longer supported by Fitbit.")]
```

Comment:
I'd love to see these enum values have integers assigned to them explicitly so removing/adding values won't negatively impact usage.

Reply:
Good call, I can add that